src/components/PopularItemCarousel/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { CarouselProvider, Slider, Slide, ButtonBack, ButtonNext, DotGroup } from 'pure-react-carousel'
+import CarouselItem from './Item'
+
+const PopularItemCarousel = () => {
+  return (
+    <CarouselProvider
+      visibleSlides={3}
+      naturalSlideWidth={400}
+      naturalSlideHeight={500}
+      totalSlides={5}
+      infinite={true}
+    >
+      <Slider>
+        <Slide index={0}>
+          <CarouselItem caption={"1st slide"} />
+        </Slide>
+        <Slide index={1}>
+          <CarouselItem caption={"2nd slide"} />
+        </Slide>
+        <Slide index={2}>
+          <CarouselItem caption={"3rd slide"} />
+        </Slide>
+        <Slide index={4}>
+          <CarouselItem caption={"4th slide"} />
+        </Slide>
+        <Slide index={5}>
+          <CarouselItem caption={"5th slide"} />
+        </Slide>
+      </Slider>
+      <ButtonBack className="buttonBack">{'<'}</ButtonBack>
```

Comment:
According to the design below, do you think we should hide this back button (`<`) when it is showing the first slide?  

<img width="479" alt="Screen Shot 2564-02-20 at 23 16 37" src="https://user-images.githubusercontent.com/32285706/108601899-afd0fc00-73d1-11eb-86d8-a0036d602c8d.png">